.gitignore

```diff
@@ -18,3 +18,4 @@ yarn-error.log*
 
 work/scripts/*
 .env
+dist/
```

Comment:
@lcfd @GnuRant Hi, please do not put dist/ in .gitignore. I know that this would be normally best practice, but in our case we need that to distribute our web components. As you might know, our store has a crawler, which takes distributed files from github directly. I will fix that, just as information for you in the future.

Reply:
@Piiit ok, good to know. I'll try to remember it in the future!